bookwyrm/templates/guided_tour/user_profile.html

```diff
@@ -0,0 +1,148 @@
+        title: "{% trans 'User Profile' %}",
+        buttons: [
+            {
+                action() {
+                    disableGuidedTour(csrf_token);
+                    return this.complete();
+                },
+                secondary: true,
+                text: "{% trans 'End Tour' %}",
+                classes: "is-danger",
+            },
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'This tab shows everything you have read towards your annual reading goal, or allows you to set one. You don\'t have to set a reading goal if that\'s not your thing!' %}",
+        title: "{% trans 'Reading Goal' %}",
+        attachTo: {
+            element: "#tour-reading-goal",
+            on: "right",
+        },
+        buttons: [
+            {
+                action() {
+                    return this.back();
+                },
+                secondary: true,
+                text: "{% trans 'Back' %}",
+            },
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'Here you can see your groups, or create a new one. A group brings together Bookwyrm users and allows them to curate lists together.' %}",
+        title: "{% trans 'Groups' %}",
+        attachTo: {
+            element: "#tour-groups-tab",
+            on: "right",
+        },
+        buttons: [
+            {
+                action() {
+                    return this.back();
+                },
+                secondary: true,
+                text: "{% trans 'Back' %}",
+            },
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'You can see your lists, or create a new one, here. A list is a collection of books that have something in common.' %}",
+        title: "{% trans 'Lists' %}",
+        attachTo: {
+            element: "#tour-lists-tab",
+            on: "right",
+        },
+        buttons: [
+            {
+                action() {
+                    return this.back();
+                },
+                secondary: true,
+                text: "{% trans 'Back' %}",
+            },
+            {
+                action() {
+                    return this.next();
+                },
+                text: "{% trans 'Next' %}",
+            },
+        ],
+    },
+    {
+        text: "{% trans 'The Books tab shows your book shelves. We\'ll explore this next.' %}",
```

Comment:
I got this text, but then the next step was to search for a book, which seems like a different next thing. Maybe "We'll explore this later."?

Reply:
Ah yes. In my mind it is "next", but we have to search for a book and add it before we can look at our book shelves, which is extremely not "next" to someone brand new to the app.